Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php

```diff
@@ -106,5 +78,42 @@ public function provideTestFixCases()
     );',
             ),
         );
+
+        $types = array('true', 'false', 'null');
+        // Equals' => true, 'NotEquals' => false); are transformed by PhpUnitStrictFixer to 'Same' and 'NotSame'
```

Comment:
Good point, I cannot recall why I removed those from the initial PR.
Update the PR with `Equals` and I've simplified the test code a bit.